### PR TITLE
[0.10.1] fix OSX environment path to allow haxe execute haxelib and neko

### PR DESCRIPTION
**Pull request by eliasku**
This fix haxe compilation & compiler code completion contribution. Allow to execute haxe/haxelib/neko on Mac OSX. Should be included to 0.10.1

### CODE REVIEW

File: src/common/com/intellij/plugins/haxe/config/sdk/HaxeSdkUtil.java [hunk 1]
```diff
@@ -22,8 +22,12 @@
 import com.intellij.execution.process.CapturingProcessHandler;
 import com.intellij.execution.process.ProcessOutput;
 import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.module.Module;
```
Comment (EBatTiVo) [inline src/common/com/intellij/plugins/haxe/config/sdk/HaxeSdkUtil.java]: Just imports?  Seems like this file just needs to be reverted.

File: src/common/com/intellij/plugins/haxe/haxelib/HaxelibProjectUpdater.java [hunk 2]
```diff
@@ -468,11 +468,11 @@ private void syncOneModule(@NotNull final ProjectTracker tracker, @NotNull Modul
           if (file != null && file.getFileType().equals(HXMLFileType.INSTANCE)) {
             PsiFile psiFile = PsiManager.getInstance(project).findFile(file);
             if (psiFile != null) {
-              Collection<HXMLClasspath> hxmlClasspaths = PsiTreeUtil.findChildrenOfType(psiFile, HXMLClasspath.class);
-              for (HXMLClasspath hxmlClasspath : hxmlClasspaths) {
-                String classpath = hxmlClasspath.getValue();
-                haxelibExternalItems.add(new HaxelibItem(classpath, VfsUtil.pathToUrl(classpath)));
-              }
+              //Collection<HXMLClasspath> hxmlClasspaths = PsiTreeUtil.findChildrenOfType(psiFile, HXMLClasspath.class);
```
Comment (EBatTiVo) [inline src/common/com/intellij/plugins/haxe/haxelib/HaxelibProjectUpdater.java]: Why comment these out for this PR?  If you want to change how the haxe libs are auto-added, (and they definitely need reworking,) I would put them into a separate PR.

Comment (eliasku) [inline src/common/com/intellij/plugins/haxe/haxelib/HaxelibProjectUpdater.java]: I will revert these changes, trapped by mistake, sorry

Comment (eliasku) [inline src/common/com/intellij/plugins/haxe/haxelib/HaxelibProjectUpdater.java]: done

File: src/common/com/intellij/plugins/haxe/config/sdk/HaxeSdkUtil.java (hunk 1 above)
Comment (eliasku) [inline src/common/com/intellij/plugins/haxe/config/sdk/HaxeSdkUtil.java]: done

**Comment by eliasku** [comment]
should fix #370
